app/src/main/res/layout/activity_main.xml

```diff
@@ -1,17 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".SplashScreenLogoAnimation.adapter.MainActivity">
+    tools:context=".MainActivity">
 
-    <androidx.recyclerview.widget.RecyclerView
+    <ListView
+        android:id="@+id/listView_showcases"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-       android:id="@+id/user_list"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        android:entries="@array/showcases"/>
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+</LinearLayout>
```

Comment:
Is there a particular reason we went with a list view instead of a recyclerview? 

I will merge this however it might be worth adding a recyclerview in future. Its easier to add animations to.